src/transport/GasTransport.cpp

```diff
@@ -839,4 +844,102 @@ void GasTransport::getBinDiffusivityPolynomials(size_t i, size_t j, double* coef
     }
 }
 
+void GasTransport::getCollisionIntegralPolynomials(size_t i, size_t j, double* astar_coeffs, 
+                                                double* bstar_coeffs, double* cstar_coeffs) const
+{   
+    for (size_t k = 0; k < (m_mode == CK_Mode ? 6 : COLL_INT_POLY_DEGREE)+1; k++) {
+        astar_coeffs[k] = m_astar_poly[m_poly[i][j]][k];
+        bstar_coeffs[k] = m_bstar_poly[m_poly[i][j]][k];
+        cstar_coeffs[k] = m_cstar_poly[m_poly[i][j]][k];
+    }
+}
+
+void GasTransport::setViscosityPolynomials(size_t i, double* coeffs)
+{
+    for (size_t k = 0; k < (m_mode == CK_Mode ? 4 : 5); k++) {
+        m_visccoeffs[i][k] = coeffs[k];
+    }
+    
+    m_visc_ok = false;
+    m_spvisc_ok = false;
+    m_viscwt_ok = false;
+    m_bindiff_ok = false;
+    m_temp = -1;
+    
+    std::cout << "Modified viscosity polynomial for species " 
```

Comment:
None of those methods should generate output.

Reply:
Those std::cout were for testing and I forgot to remove them. They are gone now.